packages/plans-grid-next/src/storybook-mocks/grid-plans.tsx

```diff
@@ -287,11 +289,91 @@ const enterprise = {
 		storageOptions: [],
 	},
 };
-export const gridPlans: GridPlan[] = [
-	freePlan,
-	personalBundle,
-	valueBundle,
-	businessBundle,
-	ecommerce,
-	enterprise,
-];
+
+export const spotlight = {
```

Comment:
I've incorporated this 'spotlight' plan into the `grid-plans.tsx` file for now, so that we have a single source of mock plans and can avoid having too many extra files for mocks eg. `grid-plan-for-spotlight.tsx`